test/integration_tests_no_travis.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+yarn run build;
+
+# Logout of clasp
+npx @google/clasp logout
+set +x
+echo "Log into clasp with a test account, or you will pollute your Apps Scripts UI with a bunch of nonsense projects."
+set -x
+npx @google/clasp login
+
+# Happy path for connector
+./test/integration-tests-no-travis/connector_happy_path.exp
+# Just check that it created a directory.
+test -d ./my-connector
+# Clean up created connector
+rm -rf ./my-connector
```

Comment:
It doesn't matter in this case, since you're deleting the directory anyway, but for future reference, you can install cleanup handlers in bash with the `trap` command:

    trap 'rm -rf ./my-connector' EXIT

That says "when this script exits for any reason, run that command on the way out."

It isn't guaranteed to run- `SIGKILL` is an instant kill- but it's a good defense-in-depth thing against Ctrl-C and other early exits.